File: unittest/scripts/devapi/scripts/connecting_to_a_session_2.js
// Assumptions: test schema exists
// Passing the paramaters in the { param: value } format
// Query the use for the user information
print("Please enter the database user information.");
var usr = prompt("Username: ", {defaultValue: "mike"});
var pwd = prompt("Password: ", {type: "password"});

// Connect to MySQL X server on network machine
var mySession = mysqlx.getSession( {
  host: 'localhost', port: 33060,
  dbUser: usr, dbPassword: pwd} );
	
var	myDb = mySession.getSchema('test');

File: unittest/scripts/py_dev_api_examples/scripts/connecting_to_a_session_2.py
# Assumptions: test schema exists
# Passing the paramaters in the { param: value } format
# Query the use for the user information
print "Please enter the database user information."
usr = shell.prompt("Username: ", {'defaultValue': "mike"})
pwd = shell.prompt("Password: ", {'type': "password"})

# Connect to MySQL X server on network machine
mySession = mysqlx.getSession( {
  'host': 'localhost', 'port': 33060,
  'dbUser': usr, 'dbPassword': pwd} )

myDb = mySession.getSchema('test')